test/unit/services/test-get-overview.js

```diff
@@ -16,7 +16,10 @@ const OVERVIEW = {
   totalPoints: 40,
   cases: '2',
   hours: '3',
-  reduction: '4'
+  reduction: '4',
+  contractedHours: 0,
```

Comment:
Should `contractedHours` returned in the results object from `getOverview` not have been replaced with the default value?